eva_submission/eload_ingestion.py

```diff
@@ -16,66 +37,84 @@ class EloadIngestion(Eload):
 
     def __init__(self, eload_number):
         super().__init__(eload_number)
-        self.eload_cfg.set(self.config_section, 'ingestion_date', value=self.now)
         self.settings_xml_file = cfg['maven_settings_file']
+        self.project_accession = self.eload_cfg.query('brokering', 'ena', 'PROJECT')
+        self.project_dir = self.setup_project_dir()
+        self.mongo_uri = get_mongo_uri_for_eva_profile('production', self.settings_xml_file)
```

Comment:
Judgment call but I am not sure hard-coding production environment is advisable here... Sometimes it may be worthwhile to test the automation in an [internal environment](https://github.com/EBIvariation/configuration/blob/22cb2ac3e675ae1913849fcaefc51bed2e2f69e1/eva-maven-settings.xml#L27) just to make sure everything works properly.

Reply:
Good point, I'll make it a configurable value.